src/js/controllers/walletHome.js

```diff
@@ -1225,44 +1226,81 @@ angular.module('copayApp.controllers').controller('walletHomeController', functi
-      if (err || lodash.isNull(feePerKb)) {
+      if (err || !lodash.isNumber(feePerKb)) {
+        self.setOngoingProcess();
         self.error = gettext('Could not get fee value');
         return;
       }
 
-      var feeToSendMaxSat = parseInt(((totalBytesToSendMax * feePerKb) / 1000.).toFixed(0));
-      if (availableBalanceSat > feeToSendMaxSat) {
-        self.lockedCurrentFeePerKb = feePerKb;
-        availableMaxBalance = strip((availableBalanceSat - feeToSendMaxSat) * self.satToUnit);
-        feeToSendMaxStr = profileService.formatAmount(feeToSendMaxSat) + ' ' + self.unitName;
-      } else {
-        self.error = gettext('Not enought funds for fee');
-        return;
-      }
+      feeToSendMaxSat = parseInt(((totalBytesToSendMax * feePerKb) / 1000.).toFixed(0));
+      availableMaxBalance = strip((availableBalanceSat - feeToSendMaxSat) * self.satToUnit);
 
-      var msg = gettextCatalog.getString("{{fee}} will be deducted for bitcoin networking fees", {
-        fee: feeToSendMaxStr
-      });
+      var opts = {};
+      opts.feePerKb = feePerKb;
+      opts.returnInputs = true;
+      var config = configService.getSync();
+      opts.excludeUnconfirmedUtxos = !config.wallet.spendUnconfirmed;
+      self.setOngoingProcess(gettextCatalog.getString('Retrieving inputs information'));
 
-      $scope.$apply();
-      confirmDialog.show(msg, function(confirmed) {
-        if (confirmed) {
-          self._doSendAll(availableMaxBalance);
-        } else {
-          self.resetForm();
+      fc.getSendMaxInfo(opts, function(err, resp) {
+        if (err) {
+          self.setOngoingProcess();
+          self.error = err;
+          $scope.$apply();
+          return;
+        }
+        if (resp.amount == 0) {
+          self.setOngoingProcess();
+          self.error = gettext("Not enought funds for fee");
+          $scope.$apply();
+          return;
+        }
+        if (availableMaxBalance == 0) {
+          self.setOngoingProcess();
+          self.error = gettext("Cannot create transaction. Insufficient funds");
+          $scope.$apply();
+          return;
+        }
+
+        var msg = gettextCatalog.getString("{{fee}} will be deducted for bitcoin networking fees", {
+          fee: profileService.formatAmount(resp.fee) + ' ' + self.unitName
+        });
+
+        self.setOngoingProcess();
+        confirmDialog.show(msg, function(confirmed) {
+          if (confirmed) {
+            self._doSendMax(resp.amount * self.satToUnit);
+            verifyExcludedUtxos();
+          } else {
+            self.resetForm();
+          }
+        });
+
+        function verifyExcludedUtxos() {
+          if (resp.utxosBelowFee > 0) {
+            var amountBelowFeeStr = profileService.formatAmount(resp.amountBelowFee) + ' ' + self.unitName;
+            self.warning = gettext("Warning: a total of " + amountBelowFeeStr + " are excluded. All they come from smaller utxos than the network fee provided");
```

Comment:
Warning is too strong. What about this:
`Note: a total of XXX were excluded. These funds come from UTXOs smaller than the network fee provided.`